classes/mock/generator.php

```diff
@@ -668,7 +668,7 @@ protected function getReturnType(\reflectionMethod $method)
         return $returnTypeCode;
     }
 
-    protected function isNullable(\reflectionType $type)
+    protected function isNullableReturnType(\reflectionType $type)
```

Comment:
yep, but it' a bit confusing. So is there any space where we can note that kind of stuff for the v4?